src/features/governance/components/Proposal.tsx

```diff
@@ -31,10 +31,10 @@ import { ProposalTransactions } from './ProposalTransactions';
 import styles from './styles.module.css';
 
 export function Proposal({ id }: { id: string }) {
-  const { proposals } = useGovernanceProposals();
+  const { proposals, isLoading } = useGovernanceProposals();
```

Comment:
a previous refactor made that `proposals` is always an empty array, so the invarient didnt work.
Regardless we should always use the `isLoading` if possible